### PR TITLE
dia2ns

### PR DESCRIPTION
**Pull request by LouisRobert2017**
i wasn't able to get the diasend 2 Nightscout thing done

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -37,6 +37,8 @@ jobs:
         run: npm install
       - name: Run Tests
         run: npm run-script test-ci
+        env:
+          NODE_OPTIONS: "--max_old_space_size=4096"
```
Comment (sulkaharo) [inline .github/workflows/main.yml:41]: Does this really require this much memory? If yes, this PR is unmergeable as most people host Nightscout in environments with less than 512 MB of RAM

**Comment by sulkaharo** [comment]
Hi! Was this PR intentional? Looks like it was made accidentally?

File: .github/workflows/main.yml (hunk 1 above)
Comment (burnedikt) [reply .github/workflows/main.yml:41]: https://github.com/nightscout/cgm-remote-monitor/pull/7820#issuecomment-1375368711

**Comment by sulkaharo** [comment]
Closing, see #7820

**Pull request closed without merging**